packages/http-cors/index.js

```diff
@@ -22,123 +69,53 @@ const defaults = {
   methods: undefined,
   origin: '*',
   origins: [],
+  enablePreflightReply: true,
```

Comment:
What do you think of `disableEarlyPreflightResponse` or `disableStaticOrigins` (name need more thought, always the hardest part) set to `true`, with it changing to `false` as the default in the next major release. We use the disable notation in other middleware and have it disabled for now so it's not a breaking change.

Reply:
i like disableEarlyPreflightResponse ... can we go with that?